language/de-DE/de-DE.com_media.ini

```
@@ -4,25 +4,46 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
 ; Note : All ini files need to be saved as UTF-8
 
+COM_MEDIA="Medien"
+COM_MEDIA_ACTION_DELETE="Datei löschen"
+COM_MEDIA_ACTION_DOWNLOAD="Datei herunterladen"
+COM_MEDIA_ACTION_EDIT="Datei ändern"
+COM_MEDIA_ACTION_PREVIEW="Dateivorschau"
+COM_MEDIA_ACTION_RENAME="Datei umbenennen"
+COM_MEDIA_ACTION_SHARE="Einen teilbaren Link erhalten"
 COM_MEDIA_ALIGN="Ausrichtung"
-COM_MEDIA_ALIGN_DESC="Dies wendet die CSS-Klassen „pull-left“, „pull-center“ oder „pull-right“ an den „<figure>“ oder „<img>“-Elementen an."
+COM_MEDIA_ALIGN_DESC="Dies wendet die CSS-Klassen 'pull-left', 'pull-center' oder 'pull-right' an den '<figure>' oder '<img>'-Elementen an."
+COM_MEDIA_BREADCRUMB_LABEL="Navigationspfad (Breadcrumb)"
 COM_MEDIA_BROWSE_FILES="Dateien suchen"
 COM_MEDIA_CAPTION="Bildbeschriftung"
+COM_MEDIA_CAPTION_CLASS_DESC="Dies wendet die CSS-Klassen 'text-left', 'text-right' oder 'text-center' am Element '<figcaption>' an."
 COM_MEDIA_CAPTION_CLASS_LABEL="Caption-Klasse"
-COM_MEDIA_CAPTION_CLASS_DESC="This will apply the entered class to the '<figcaption>' element. For example: 'text-left', 'text-right', 'text-center'"
 COM_MEDIA_CLEAR_LIST="Liste löschen"
 COM_MEDIA_CONFIGURATION="Medien: Optionen"
+COM_MEDIA_CONFIRM_DELETE_MODAL="Löschen"
+COM_MEDIA_CONFIRM_DELETE_MODAL_HEADING="Löschen bestätigen"
 COM_MEDIA_CREATE_FOLDER="Verzeichnis anlegen"
+COM_MEDIA_CREATE_NEW_FOLDER="Neues Verzeichnis anlegen"
+COM_MEDIA_CREATE_NEW_FOLDER_ERROR="Fehler beim Anlegen des Verzeichnis."
+COM_MEDIA_CREATE_NEW_FOLDER_SUCCESS="Verzeichnis angelegt."
 COM_MEDIA_CURRENT_PROGRESS="Aktueller Fortschritt"
+COM_MEDIA_DECREASE_GRID="Rastergröße verkleinern"
+COM_MEDIA_DELETE_ERROR="Fehler beim Löschen der Datei."
+COM_MEDIA_DELETE_SUCCESS="Datei gelöscht."
 COM_MEDIA_DESCFTP="Zum Hochladen, Ändern und Löschen von Mediendateien, benötigt Joomla! FTP-Kontodaten. Bitte diese in die unteren Formularfelder eingeben."
 COM_MEDIA_DESCFTPTITLE="FTP-Kontodaten"
 COM_MEDIA_DETAIL_VIEW="Detailansicht"
 COM_MEDIA_DIRECTORY="Verzeichnis"
 COM_MEDIA_DIRECTORY_UP="Übergeordnetes Verzeichnis"
+COM_MEDIA_DROP_FILE="Verschieben um Datei(en) hochzuladen"
```

hmm hier geht es um Drag-and-Drop

```suggestion
COM_MEDIA_DROP_FILE="Datei per Drag-and-Drop hochladen"
```